src/ring_theory/dedekind_domain/ideal.lean

```diff
@@ -855,3 +855,165 @@ end
 end is_dedekind_domain
 
 end height_one_spectrum
+
+section chinese_remainder
+
+open ideal unique_factorization_monoid
+open_locale big_operators
+
+variables {R}
+
+lemma ring.dimension_le_one.prime_le_prime_iff_eq (h : ring.dimension_le_one R)
+  {P Q : ideal R} [hP : P.is_prime] [hQ : Q.is_prime] (hP0 : P ≠ ⊥) :
+  P ≤ Q ↔ P = Q :=
+⟨(h P hP0 hP).eq_of_le hQ.ne_top, eq.le⟩
+
+variables {R} [is_domain R]
```

Comment:
You can move this line after `ideal.coprime_of_no_prime_ge`, that doesn't need anything different. Also, `R` is already implicit.